### PR TITLE
feature: add footer, fix card margins, refactor HTML

### PR DESCRIPTION
**Pull request by jaketae**
This PR addresses the following:

* Introduce a simple footer with "Back to Top" feature for easier site navigation
* Include `footer.html` across the board for consistent GA tracking
* Fix CSS issues with Workshop and Socials cards to remove visible margins
* Reindent HTML files for better code readability and maintainability (There were some files where tags were not closed or mismatched.) For an accurate diff, refer to this [link](https://github.com/ICLR/iclr.github.io/pull/47/files?w=1). 

Attached are screenshots of the footer and the problematic cards, before and after, for reference.

Footer:
Screen Shot 2020-04-23 at 2 14 17 AM

Before:
Screen Shot 2020-04-23 at 2 13 55 AM

After:
Screen Shot 2020-04-23 at 2 13 47 AM

### CODE REVIEW

**Comment by jaketae** [comment]
Hello @srush, I was just wondering if it would be possible for you to review this PR, that is only if your time permits. I realized that a lot of merge conflicts are happening due to rapid development and the subsequent updates being made to the code base.

But of course, this is most likely not at the top of the priority list, and it is absolutely not my intent to bother you for a code review. I totally understand if this gets pushed down along the way. 

Thank you for giving volunteers the opportunity to contribute! I hope you have a nice day.

**Comment by srush** [comment]
Thanks Jake! This is really nice.

**Comment by srush** [comment]
What formatter are you using?

File: templates/pages/index.html [hunk 1]
```diff
@@ -22,170 +20,141 @@
...
+            <h2 class="text-center"> Sponsors </h2>
           </div>
         </div>
+        <!-- <center> -->
+        <!-- <img src="static/images/sponsors.png" width="100%" /> -->
+        <!-- </center> -->
+        {% for section in sponsors %}
+        <div class="row p-3">
+          <div class="col-12 bd-content ">
+            <h3 class="">
```
Comment (jaketae) [inline templates/pages/index.html:102]:
```suggestion
            <h3>
```

File: templates/pages/index.html [hunk 2]
```diff
@@ -22,170 +20,141 @@
...
+            {{section.tier}} </h2>
```
Comment (jaketae) [inline templates/pages/index.html:103]:
```suggestion
            {{section.tier}} </h3>
```

**Comment by jaketae** [comment]
Hello @srush, thank you for the quick response and merge, I appreciate it. As for the formatter, I'm using a Sublime Text plugin called [HTML-CSS-JS Prettify](https://packagecontrol.io/packages/HTML-CSS-JS%20Prettify), with some minor custom settings. Not sure how this information would translate into your dev environment, but hope it helps nonetheless!

**Pull request merged**